.github/workflows/ruby.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       PGUSER: postgres
       # Rails verifies the time zone in DB is the same as the time zone of the Rails app
       TZ: "Europe/Warsaw"
+      MAPBOX_API_KEY: pk.eyJ1IjoibWF0aGlhczIzNCIsImEiOiJja3U1N2o0MGoybGl5MnVxaDcyaHdjZDFpIn0.S-c3HtdhqhyAR6izLxY-iQ
```

Comment:
This is a public key so it should be fine to expose it here, the key will be exposed in our html normally anyway.